lib/ansible/plugins/lookup/genmac.py

```diff
@@ -0,0 +1,51 @@
+# (c) 2018, Abdoul Bah (@helldorado) <bahabdoul at gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: genmac
+    author:
+      - Abdoul Bah (@helldorado) <bahabdoul(at)gmail.com>
+    version_added: "2.7"
+    short_description: Generates a MAC address with the defined OUI prefix
+    description:
+      - This lookup return a random mac address.
+    options:
+      _terms:
+        description: OUI prefix. See http://standards-oui.ieee.org/oui/oui.txt
+        default: 'AC:DE:48'
+      format:
+        description: Convert MAC address to given format
```

Comment:
Where does a user find out what the formats are?

Reply:
Indeed, I will update the doc.